resources/js/components/app-setting.js

```diff
@@ -0,0 +1,59 @@
+// myScript.js
+window.showPopup = function () {
+  document.getElementById("popup").style.display = "block";
+  loadSVGIcons(); // Load icons when the popup is shown
+}
+
+window.hidePopup = function () {
+  document.getElementById("popup").style.display = "none";
+}
+
+function loadSVGIcons() {
```

Comment:
right click function name -> AI Actions > Write Documentation for each class and function/method in this pull request. After AI generates the DocBlock, edit to make sure the documentation it adds makes sense.

Reply:
![Screenshot from 2024-02-23 11-08-04](https://github.com/thecodezone/dt-home/assets/5910297/af59e3af-c3b6-41c4-9ecd-f8f1bb9bbbda)